Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml

```diff
@@ -2957,6 +2956,46 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: e8aecf09-e057-48c2-8fcb-d67de5afbc91
+    type: regular
+    task:
+      id: e8aecf09-e057-48c2-8fcb-d67de5afbc91
+      version: -1
+      name: Read QR Code from image files
```

Comment:
Recommend capitalization fix (in this file, task names do not seem to be all capitalized):
Read QR code from image files